### ISSUE TITLE
Add https://login.live.com/ in the issue flowchart

### ISSUE THREAD

**Issue by eibma** [EionRobb/skype4pidgin#516]
There seems to be a state for some accounts where the web.skype.com login is working, but the login.live.com page is requesting some account merging functionality for other microsoft products. In this state I just get the "Failed getting Magit T value" error. Please add a login on https://login.live.com in the flowchart.

And thanks for your effort on this Project!

**Comment by xvitaly** [comment]
> Please add a login on https://login.live.com in the flowchart.

2FA login is not supported by plugin.

**Comment by eibma** [comment]
Yes, I do understand that. As far as I know it was not configured on my account. But I could not login until I completed my profile on login.live.com. I'm afraid I don't have any logs preserved but I've noticed in the logs that requests to login.live.com have been made. It was just not obvious to me why I cannot login using the plugin. If it would have been in the flow chart I might have tried to login on login.live.com earlier. 

Feel free to close this issue if you think it is not necessary.

regards
